tests/atlocal.in

```diff
@@ -50,6 +50,20 @@ else
     CAP_DISABLED=true;
 fi
 
+# Enable malloc-check as a cheap way to find some use-after-free and
+# uninitialized read problems when using glibc, and doesn't affect
+# normal operation or other libc.
+# Only for GLIBC >=2.34
```

Comment:
This seems like an omission from the previous version of this patch that was supposed to check for the GLIBC version.